packages/components/src/DateTimePickers/DateTime/datetime-extraction.js

```diff
@@ -88,19 +62,21 @@ function timeToSeconds(hours, minutes, seconds) {
  * Set the time to the provided date
  * @param date {Date} Date in current TZ
  * @param time {{hours: string, minutes: string, seconds: string}} Time in current TZ
- * @param useUTC {boolean} Indicates that we ask for a date in UTC TZ
  * @returns {Date}
  */
-function dateAndTimeToDateTime(date, time, { useUTC }) {
+function dateAndTimeToDateTime(date, time) {
 	if (date === undefined || time === undefined) {
 		return INTERNAL_INVALID_DATE;
 	}
 
 	try {
+		if (typeof time === 'string') {
+			// eslint-disable-next-line no-param-reassign
+			time = strToTime(time);
+		}
 		const { hours, minutes, seconds } = time;
 		const timeInSeconds = timeToSeconds(hours, minutes, seconds);
-		const localTimezoneDate = setSeconds(date, timeInSeconds);
-		return useUTC ? convertToUTC(localTimezoneDate) : localTimezoneDate;
```

Comment:
Why have you removed the utc thing ? 
Even if you deprecate it, you must maintain the behavior, or it's a breaking change. Then when we want to remove it in a major version, we can remove the behavior

Reply:
I thought date is already converted to utc when pass from date picker to datetime picker, so I just add seconds base on date.
But you're right, better to handle utc/timezone in datetime manager, since it may result in different date for local and target timezone.